pkg/api/handlers/version.go

```diff
@@ -30,7 +35,52 @@ func (v *VersionHandler) getVersionResponse() map[string]string {
 			values["error"] = err.Error()
 		}
 	}
-	values["otto"] = version.Get().String()
+	values["acorn"] = version.Get().String()
```

Comment:
This is going to change to `obot` soon, but I figured updating this to follow the current branding will make that easier.